README.adoc

```diff
@@ -50,6 +54,21 @@ For the full documentation of the Guillotine project, visit the Enonic Developer
 
 == Breaking changes
 
+=== Breaking changes in 5.0
+
+- Type "Attachment" is returned instead of "String" for form-item with type "AttachmentUploader"
+- Type "Content" fields "dataAsJson" and "pageAsJson" return "JSON" instead of "String"
+- Fields "config" of the "PageComponentData", "LayoutComponentData" and "PartComponentData" types return "JSON" instead of "String"
+- Type "Subscription": Field "newEvent" renamed to "event". The "type" and "localOnly" arguments are removed
+- Type "UrlTypeType" renamed to "UrlType"
+- Type "HeadlessCms": Field "query" returns the "QueryResult" type instead of an array of the "Content" type
+- Type "HeadlessCms": Field "queryConnection" returns the "QueryContentConnection" type instead of an array of the "ContentConnection" type
+- Generation of field names for "OptionSet" and "ItemSet" types do not use camel-case anymore and were uniformed with the rest of field naming
+- Type "TextComponentData": Field "value" returns the "HtmlEditorResult" type instead of the "String" type and supports the `processHtml` argument with the "ProcessHtmlInput" type.
```

Comment:
HtmlArea change is non-breaking, or is it?

Reply:
done